.dependabot/config.yml

```diff
@@ -7,20 +7,20 @@ update_configs:
       - match:
           update_type: all
   - package_manager: "python"
-    directory: "/amundsencommon"
+    directory: "/common"
     update_schedule: "monthly"
   - package_manager: "python"
-    directory: "/amundsendatabuilder"
+    directory: "/databuilder"
```

Comment:
why are we changing the name? also databuilder points to `amundsendatabuilder` root dir?

Reply:
Note that this PR is against the monorepo branch. We're changing the name of the in-repo folders to remove the redundancy, but not the pypi package names or docker image. I cleared this with the Lyft team, they were able to capture those changes at the same time